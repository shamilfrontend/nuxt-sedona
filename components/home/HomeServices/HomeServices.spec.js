import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('HomeServices', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {})
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
