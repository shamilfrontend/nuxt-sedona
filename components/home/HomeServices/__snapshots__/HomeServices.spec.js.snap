// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeServices should match snapshot 1`] = `
<section class="home-services">
  <div class="home-services__item home-services__item_house">
    <h4 class="home-services__title">Жильё</h4>
    <p class="home-services__text">
      Рекомендуем пожить в настоящем мотеле, всё как в кино!
    </p>
  </div>
  <div class="home-services__item home-services__item_food">
    <h4 class="home-services__title">Еда</h4>
    <p class="home-services__text">
      Всегда заказывайте фирменный бургер, вы не разочаруетесь!
    </p>
  </div>
  <div class="home-services__item home-services__item_souvenir">
    <h4 class="home-services__title">Сувениры</h4>
    <p class="home-services__text">
      Не только китайского, но и местного производства!
    </p>
  </div>
</section>
`;
